doc/rpc/pathfinder_ws.json

```diff
@@ -87,6 +123,17 @@
     "components": {
         "contentDescriptors": {},
         "schemas": {
+            "SUBSCRIPTION_EVENT": {
+                "title": "Subscription event",
+                "oneOf": [
+                    {
+                        "$ref": "#/components/schemas/BLOCK_HEADER"
+                    },
+                    {
+                        "$ref": "#/components/schemas/EMITTED_EVENT"
+                    }
+                ]
+            },
```

Comment:
Perfect. But you need to update `pathfinder_subscription` to fire this - currently its still only `BLOCK_HEADER` explicitly there.